utility/README.md

```diff
@@ -0,0 +1,190 @@
+# Utility Module
+
+This document is meant to be a placeholder to serve as a living representation of the parent 'Utility Module' codebase until the code matches the [1.0 Utility Module specification](https://github.com/pokt-network/pocket-network-protocol/tree/main/utility).
+
+The spirit of the documentation is to continuously update and inform the reader of the general scope of the Utility Module as breaking, rapid development occurs.
+
+_This document will be archived once the implementation is synonymous with the current 1.0 Utility specification, ._
+
+# Origin Document
+
+Currently, the Utility Module minimally implements the first iteration of an account based, state machine, blockchain protocol.
+
+Synonymous technologies to this current iteration are 'MVP proof of stake' blockchain state machines, that are NOT Pocket Network specific.
+
+The current implementation does add the fundamental Pocket Network 1.0 actors:
+
+- Accounts
+- Validators
+- Fishermen
+- Applications
+- Service Nodes
+
+And implement the basic transaction functionality:
+
+- Send-Tx
+- Stake
+- Unstake
+- EditStake
+- Pause
+- Unpause
+
+And a few additional skeleton implementations for pocket specific transactions:
+
+- FishermanPauseServiceNode [x] Implemented
+- TestScore [x] Shell // requires sessions & report card structures
```

Comment:
I don't fully understand the syntax for `TestScore [x] Shell`

Reply:
Can you call it a placeholder / skeleton / wrapper, whenever I see shell, I think: https://en.wikipedia.org/wiki/Shell_(computing)

Reply:
Yes - will do